chart/README.md

```diff
@@ -40,7 +40,7 @@ cluster using the [Helm](https://helm.sh) package manager.
 * Supported AWS executors with AWS provider version ``8.21.0+``:
    * ``airflow.providers.amazon.aws.executors.batch.AwsBatchExecutor``
    * ``airflow.providers.amazon.aws.executors.ecs.AwsEcsExecutor``
-* Supported Airflow version: ``1.10+``, ``2.0+``
+* Supported Airflow version: ``1.10+``, ``2.0+``, ``3.0+``
```

Comment:
How long do we keep 1.10 support?

Reply:
We should drop them in Helm Chart 2.0

Reply:
Timing wise -- will let Jed take a call on that once he is back